algorithms/ar-bsrh/csharp/BinarySearch.cs

```diff
@@ -0,0 +1,45 @@
+// C# implementation of Binary Search 
+using System; 
+
+class BinarySearch { 
+	// Returns index of x if it is present in arr[], 
+	// else return -1 
+	static int DoBinarySearch(int[] arr, int x) 
+	{ 
+		int l = 0, r = arr.Length - 1; 
+		while (l <= r) {
+
+			int m = l + (r - l) / 2; //mid point
+
+			//1) Check if x is present at mid 
+			if (arr[m] == x) 
+				return m; 
+
+			//2) If x greater, ignore left half 
+			if (arr[m] < x) 
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6330839)